controlpanel/frontend/jinja2/tools.html

```diff
@@ -7,13 +7,14 @@
 
 <h1 class="govuk-heading-xl">Your tools</h1>
 
-<p class="govuk-body">The status of your tools will update automatically.</p>
+{# <p class="govuk-body">The status of your tools will update automatically.</p>
```

Comment:
Sorry for the silly jinja question: What do the hashes (`#`) do? Comment the stuff between them?

Reply:
Yep